x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/IndexPrivilege.java

```diff
@@ -48,6 +48,8 @@
             ClusterSearchShardsAction.NAME);
     private static final Automaton CREATE_AUTOMATON = patterns("indices:data/write/index*", "indices:data/write/bulk*",
             PutMappingAction.NAME);
+    private static final Automaton CREATE_DOC_AUTOMATON = Automatons.minusAndMinimize(CREATE_AUTOMATON,
+        patterns("indices:data/write/index:op_type/index"));
```

Comment:
I would rather that this automaton be whitelist rather than blacklist.
That is, rather than excluding `op_type/index` can we just explicitly allow `op_type/create`